client/src/components/Cards/Cards.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Card from '../Card/Card'
+
+import { nanoid } from 'nanoid'
+
+const sampleCards = [
+  { text: 'What is node?', author: 'Stefan Bering' },
+  { text: 'What is React?', author: ' Timur Buchberger' },
+  { text: 'What is styled components?', author: ' Danilo Bräuer' },
+  { text: 'What is node?', author: 'Stefan Bering' },
+  { text: 'What is React?', author: ' Timur Buchberger' },
+  { text: 'What is styled components?', author: ' Danilo Bräuer' },
+  { text: 'What is node?', author: 'Stefan Bering' },
+  { text: 'What is React?', author: ' Timur Buchberger' },
+  { text: 'What is styled999 components?', author: ' Danilo Bräuer' },
+]
+
+const Cards = () => {
+  const [data, setData] = useState(sampleCards)
+
+  return (
+    <StyledMain>
+      {data.map(card => {
+        console.log(card)
+        return <Card text={card.text} author={card.author} key={nanoid()} />
```

Comment:
Attention! Please don't create a new `id` as key _with every render_! This will mess up React and will lead to problems! Right now, every time this component renders, a new `id` is created, therefore the `key` will _always_ be different and React can't compare. Add the `id` to your mock data. @StefanBehring @timurbuch @codeNilo93

Reply:
Didn't even think about that. Will change with the next commit.